### ISSUE TITLE
Plugin v0.1.0 doesn't work for me with Telerik platform

### ISSUE THREAD

**Issue by abhayastudios** [NativeScript/push-plugin#90]
Hi,

I am trying to use the push plugin with Telerik platform. At the end of my login procedure (includes authenticating the user with Telerik platform) I call the method below. After reading a little here I moved the location of the require to the exact spot where the plugin is actually used:

```
public registerPushDevice() {
    var pushPlugin = require("nativescript-push-notifications");

    /* register pushPlugin with Telerik platform */

    this.telerik.push.register(this.devicePushSettings, () => {
      console.log("Successful registration in Telerik Platform. You are ready to receive push notifications.");
    }, (error) => {
      console.log('Failed to register push device in Telerik Platform');
      console.dump(error);
    });
  }
```

This results in in an error `{ "code": "MODULE_NOT_FOUND" }`. The thing is that AFAIK the pushNotification module is found properly, because if I register with firebase directly it works just fine:

```
public registerPushDevice() {
    var pushPlugin = require("nativescript-push-notifications");

    /* register pushPlugin with Firebase directly */

    pushPlugin.register({ senderID: this.config.telerik.googleFirebaseSenderId }, (data) => {
      console.log("Successful registration in Firebase. You are ready to receive push notifications.");
    }, (error) => {
      console.log('Failed to register push device in Firebase');
      console.dump(error);
    });
}
```

Any idea what might be going on? I already updated the everlive-sdk to the latest v1.9.2 but it didn't help either. I did update the `build.gradle` and `google-services.json` according to the docs (I guess otherwise the Firebase registration would not work). Also I removed and re-added the android platform to rule out any issues with that.

Thanks!

**Comment by AntonDobrev** [comment]
Answered in - http://www.telerik.com/forums/nativescript-push-plugin-not-working-with-telerik-platform

**Comment by AntonDobrev** [comment]
Fixed in [0.1.1](https://github.com/NativeScript/push-plugin/commit/ca94e021560c0a7d9bb0db95a703478418e70bf3).

**Comment by abhayastudios** [comment]
Indeed working! Thanks so much @AntonDobrev :)
